.github/CODEOWNERS

```diff
@@ -21,7 +21,14 @@ ddtrace/internal/codeowners.py      @DataDog/apm-core-python @datadog/ci-app-lib
 
 # Debugger
 ddtrace/debugging/                  @DataDog/debugger-python
+ddtrace/settings/dynamic_instrumentation.py     @DataDog/debugger-python
+ddtrace/internal/injection.py       @DataDog/debugger-python @DataDog/apm-core-python
+ddtrace/internal/wrapping.py        @DataDog/debugger-python @DataDog/apm-core-python
```

Comment:
👍 for co-ownership since these modules are being used for wrapping/instrumenting integrations